src/facebook/facebookdatatypesyncadaptor.cpp

```diff
@@ -220,7 +220,7 @@ void FacebookDataTypeSyncAdaptor::signOnError(const SignOn::Error &error)
 
     // if the error is because credentials have expired, we
     // set the CredentialsNeedUpdate key.
-    if (error.type() == SignOn::AuthSession::UserInteractionError) {
+    if (error.type() == SignOn::Error::UserInteraction) {
         setCredentialsNeedUpdate(account);
```

Comment:
Is the first one, used before here, a error coming from a failure in signon-ui (if one exists) ?